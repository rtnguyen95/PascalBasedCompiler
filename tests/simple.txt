float fahr;
float upper;
float a;
while (fahr < upper) do a = 23.00; whileend