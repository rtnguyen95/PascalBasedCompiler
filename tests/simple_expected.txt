TOKENS        Lexemes

KEYWORD     = while
SEPARATOR   = (
IDENTIFIER  = fahr
OPERATOR    = <
IDENTIFIER  = upper
SEPARATOR   = )
IDENTIFIER  = a
OPERATOR    = =
FLOAT       = 23.00
KEYWORD     = whileend
